app/metrics.yaml

```diff
@@ -6954,3 +6954,39 @@ search_terms:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-12-01"
+
+wallpapers:
+  discovered_wallpaper_feature:
+    type: boolean
+    lifetime: application
```

Comment:
@rocketsroger Does a `application` lifetime make sense here? That way the frequency of reports is a little lower. [slack thread](https://mozilla.slack.com/archives/C4D5ZA91B/p1643142448136500) for context

Reply:
From what I read in [here](https://mozilla.github.io/glean/book/user/metrics/adding-new-metrics.html?highlight=lifetime#a-lifetime-example), would this value be cleared when the application is restarted?

Reply:
I think "cleared" in this context means that it becomes untracked by Glean, so that if it is `track`ed again it will be resent.

Reply:
So we would send this everytime the app is restarted and the user "discovered" the wallpaper feature?

Reply:
Yep! Based on the slack thread above, it sounds like that's what DS would prefer since we're looking to avoid adding a shared pref

Reply:
Hmm, apparently my understand there was not correct. I'm seeing the `DebugMetricController` report the metric every time I click the logo to switch wallpapers still.